src/main/java/com/CezaryZal/api/day/model/ObjectToSaveDay.java

```diff
@@ -0,0 +1,63 @@
+package com.CezaryZal.api.day.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.time.LocalDate;
+
+@ToString
+@Getter
+@NoArgsConstructor
+public class ObjectToSaveDay{
```

Comment:
Czy ten obiekt jest potrzebny? Czy to nie jest fragment DayDto? Czy do update obiektu nie można skorzystać z DayDto?

Reply:
Nad tym wcześniej się zastanawiałem.
Teoretycznie można do tego wykorzystać Dto, jednak chciałem rozbić te operacje na oddzielne obiekty, głównie ze względu na klienta.
Ewentualnie jak zrobię validację inputu mógłbym ograniczyć się tylko do Dto.